docs/docs/apis-extras/timeouts.md

```
@@ -9,17 +9,18 @@ sidebar_position: 2
 This API lets you set the request timeout for the Ollama client.
 
 ```java
+import io.github.ollama4j.Ollama;
 import io.github.ollama4j.OllamaAPI;
 
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)